From 206efbf7f6936c00f252d3fdf9af21edcd0f412d Mon Sep 17 00:00:00 2001
From: Ole Streicher <olebole@debian.org>
Date: Sat, 11 Mar 2023 12:54:21 +0100
Subject: [PATCH 6/9] Don't force the use of gcc

This makes the C compiler configurable during build process with the
CC environment variable.
---
 unix/f2c/libf2c/makefile.u | 2 +-
 unix/f2c/src/makefile.u    | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/unix/f2c/libf2c/makefile.u b/unix/f2c/libf2c/makefile.u
index 6d05dc6ed..33d2e9c10 100644
--- a/unix/f2c/libf2c/makefile.u
+++ b/unix/f2c/libf2c/makefile.u
@@ -13,7 +13,7 @@
 # to the CFLAGS = line below.
 
 .SUFFIXES: .c .o
-CC = cc
+CC ?= cc
 SHELL = /bin/sh
 CFLAGS = -O -w $(HSI_CF)
 
diff --git a/unix/f2c/src/makefile.u b/unix/f2c/src/makefile.u
index 0e2c73519..f4a4f1c56 100644
--- a/unix/f2c/src/makefile.u
+++ b/unix/f2c/src/makefile.u
@@ -1,7 +1,7 @@
 #	Makefile for f2c, a Fortran 77 to C converter
 
 .SUFFIXES: .c .o
-CC = cc
+CC ?= cc
 CFLAGS = -O -w $(HSI_CF)
 LDFLAGS = $(HSI_LF)
 SHELL = /bin/sh
-- 
2.39.2

